h/feeds/atom.py

```diff
@@ -28,8 +29,8 @@ def _feed_entry_from_annotation(annotation, annotation_url, annotation_api_url=N
         ),
         "author": {"name": name},
         "title": annotation.title,
-        "updated": _utc_iso8601_string(annotation.updated),
-        "published": _utc_iso8601_string(annotation.created),
+        "updated": utc_iso8601(annotation.updated),
+        "published": utc_iso8601(annotation.created),
         "content": annotation.description,
```

Comment:
Oh no... the atom and RSS feeds are presenters in disguise. I might pretend I've not seen this.